### ISSUE TITLE
System.Net.Mail.Attachment takes ownership of stream

### ISSUE THREAD

**Issue by Bouke** [DotNetAnalyzers/IDisposableAnalyzers#217]
`System.Net.Mail.Attachment(Stream, String)` shouldn't raise IDISP001:

```csharp
new System.Net.Mail.Attachment(new MemoryStream(), string.Empty);
```

**Comment by JohanLarsson** [comment]
Thanks for reporting!
Never used that API, adding spexial case.
